package.json

```diff
@@ -10,17 +13,21 @@
     "rollup-plugin-svelte": "^5.0.3",
     "rollup-plugin-terser": "^4.0.4",
     "sirv-cli": "^0.4.4",
-    "svelte": "^3.0.0"
+    "svelte": "^3.0.0",
+    "svelte-loader": "^2.13.6"
   },
   "dependencies": {
+    "d3-format": "^1.4.1",
     "firebase": "^6.3.0",
+    "flexsearch": "^0.6.30",
     "plotly.js-basic-dist": "^1.48.3"
```

Comment:
We could take firebase and plotly out of here if you're not using them.

Reply:
good point